src/openforms/config/models.py

```diff
@@ -329,6 +329,28 @@ class GlobalConfiguration(SingletonModel):
         null=True,
         help_text=_("The 'idsite' of the website you're tracking in Piwik."),
     )
+    piwik_pro_url = models.CharField(
+        _("Piwik PRO server URL"),
+        max_length=255,
+        blank=True,
+        help_text=_(
+            "The base URL of your Piwik PRO server, e.g. 'https://your-instance-name.piwik.pro/'."
+        ),
+    )
+    piwik_pro_site_id = models.CharField(
+        _("Piwik PRO site ID"),
+        blank=True,
+        max_length=255,
+        help_text=_(
+            "Typically looks like XXX-XXX-XXX-XXX-XXX. The 'idsite' of the website you're tracking in Piwik PRO."
+        ),
+        validators=[
+            RegexValidator(
+                regex=r"(\w+-){4}\w+",
+                message=_("The site ID must be a valid Piwik PRO site ID."),
+            ),
+        ],
```

Comment:
Is this always three characters per group, or does it vary? If it's a fixed length, the `max_length` should reflect this and the regex pattern should specify the number of characters.

Reply:
Not fully sure if this is the case. I will revise before making any changes.

Reply:
Indeed it can be made more specific to suit piwik pro's standards. Addressed in https://github.com/open-formulieren/open-forms/pull/1720/commits/f6d46bdc4ea1f9205e45df167bdcf2a6a8f0fb95